fsutils/libtinycbor/.gitignore

```diff
@@ -0,0 +1,2 @@
+/minmea
```

Comment:
why add this

Reply:
I used minmea was base, so I missed that, let me fix it, thank you!